packages/modules/viewer-react/src/components/iModel/IModelViewer.tsx

```diff
@@ -40,7 +40,10 @@ export const IModelViewer: React.FC<ModelProps> = ({
     // set the active frontstage to the current default
     if (defaultFrontstage) {
       // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      FrontstageManager.setActiveFrontstageDef(defaultFrontstage.frontstageDef);
+      setImmediate(
```

Comment:
unfortunately, we cant use this. Lack of browser support and no plans to standardize:

https://caniuse.com/setimmediate

https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate